install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -35,11 +35,11 @@ spec:
               echo "Gitpod: Perform the check"
               while [ "$(helm status -n {{repl Namespace }} cert-manager -o json | jq '.info.status == "deployed"')" = "false" ];
               do
-                echo "Gitpod: Release not found - will retry in 10s"
+                echo "Gitpod: cert-manager release not found - will retry in 10s"
```

Comment:
Personally, I would remove the whole `initContainer` section. Gitpod installer will handle having no cert-manager by erroring, which is probably an easier way to debug.

If we did keep this in, this command would only check in the namespace that Gitpod will be installed it - it's likely (but not guaranteed) to be in the `cert-manager` namespace.

Reply:
Agree! Removing this!